src/app/organizations/manage/policies.component.ts

```diff
@@ -158,13 +102,11 @@ export class PoliciesComponent implements OnInit {
         this.orgPolicies.forEach(op => {
             this.policiesEnabledMap.set(op.type, op.enabled);
         });
-        this.policies.forEach(p => {
-            p.enabled = this.policiesEnabledMap.has(p.type) && this.policiesEnabledMap.get(p.type);
-        });
+
         this.loading = false;
     }
 
-    edit(p: any) {
+    edit(p: BasePolicy) {
```

Comment:
I know you didn't name it, but I'd rather see a clearer name for this param.

```suggestion
    edit(policy: BasePolicy) {
```